Intersect.Client/Networking/PacketHandler.cs

```diff
@@ -1014,10 +1014,30 @@ public void HandlePacket(IPacketSender packetSender, EntityAttackPacket packet)
                 return;
             }
 
-            if (attackTimer > -1 && en != Globals.Me)
+            if (!packet.IsBlocking)
             {
-                en.AttackTimer = Timing.Global.Ticks / TimeSpan.TicksPerMillisecond + attackTimer;
-                en.AttackTime = attackTimer;
+                //attack handler
+                if (attackTimer > -1 && en != Globals.Me)
+                {
+                    en.AttackTimer = Timing.Global.Ticks / TimeSpan.TicksPerMillisecond + attackTimer;
+                    en.AttackTime = attackTimer;
+                }
+            }
+            else
+            {
+                //blocking handler
+                if(en == Globals.Me)
+                {                    
+                    if (attackTimer > -1)
+                    {
+                        en.AttackTimer = Timing.Global.Ticks / TimeSpan.TicksPerMillisecond + attackTimer;
+                        en.IsBlocking = true;
+                    }
+                    else
+                    {
+                        en.IsBlocking = false;
+                    }
+                }
```

Comment:
I think this can be greatly simplified to something like:

```csharp
var isSelf = en == Globals.Me;
if (isSelf)
{
    en.IsBlocking = packet.IsBlocking;
}

if (attackTimer > -1)
{
    en.AttackTimer = Timing.Global.Ticks / TimeSpan.TicksPerMillisecond + attackTimer;
    if (!isSelf)
    {
        en.AttackTime = attackTimer;
    }
}
```